packages/Sortable/tests/Sortable.cypress.js

```diff
@@ -25,7 +25,7 @@ describe("<Sortable />", () => {
       .trigger("mousemove", { button: 0, ...dragOffset })
       .wait(animationDelay);
     cy.get(selector.root)
-      .trigger("mousemove", { button: 0, ...dragOffset })
+      .trigger("mousemove", { button: 0, ...dragOffset, clientY: 100 })
```

Comment:
If this is really needed, maybe it makes more sense on line 13?

Reply:
But doing that would affect the other tests.  After the upgrade, all the other tests still worked... this one just started failing until I added this.

Reply:
Just tried, doesnt work.  Has to be on this line only 🤷‍♂